### PR TITLE
fix: use correct auth keys for Facebook and Twitter login

### LINKED ISSUE #201: ParseFacebook sets accessToken instead of access_token
Hi!

Unless I'm doing something wrong, I'm unable to login with Facebook.

The issue is cause by the fact that Parse Server uses `access_token`, yet Parse Swift uses `accessToken`.

Failing test:

```
func testMakeDictionary() {
        let authData = ParseFacebook<User>
            .AuthenticationKeys.id.makeDictionary(userId: "testing",
                                                  accessToken: "testAccessToken",
                                                  authenticationToken: "testAuthToken")
        XCTAssertEqual(authData["id"],"testing")
        XCTAssertEqual(authData["access_token"],"testAccessToken")

        let secondAuthData = ParseFacebook<User>
            .AuthenticationKeys.id.makeDictionary(userId: "testing",
                                                  accessToken: nil,
                                                  authenticationToken: "testAuthToken")

        XCTAssertEqual(secondAuthData["id"],"testing")
        XCTAssertEqual(secondAuthData["token"],"testAuthToken")
    }
```

https://github.com/parse-community/parse-server/blob/9ea355b4635226ae4da17c8cc5fb0321e3fdec5e/src/Adapters/Auth/facebook.js#L24

### PR DESCRIPTION
**Pull request by dblythy**
Closes #201.

Sets correct payload for FB login.

### CODE REVIEW

**Comment by cbaker6** [comment]
Though this will work, it uncovers a deeper issue. The Keys should be encoded instead of using the coding keys directly

File: Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift [hunk 1]
```diff
@@ -44,19 +44,19 @@ public struct ParseFacebook<AuthenticatedUser: ParseUser>: ParseAuthentication {
                             authenticationToken: String?,
                             expiresIn: Int? = nil) -> [String: String] {
 
-            var returnDictionary = [AuthenticationKeys.id.rawValue: userId]
+            var returnDictionary = [CodingKeys.id.rawValue: userId]
```
Comment (cbaker6) [inline Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift:47]: Please revert all of these back to AuthenticationKeys

Comment (cbaker6) [reply Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift:47]: The actual fix should be applied to lines 25-27:

```swift
enum AuthenticationKeys: String, Codable {
        case id // swiftlint:disable:this identifier_name
        case authenticationToken = "token"
        case accessToken = "access_token"
        case expirationDate = "expiration_date"
```

You can deleted the CodingKeys, 29-34

File: Tests/ParseSwiftTests/ParseFacebookCombineTests.swift [hunk 2]
```diff
@@ -84,7 +84,22 @@ class ParseFacebookCombineTests: XCTestCase { // swiftlint:disable:this type_bod
         #endif
         try ParseStorage.shared.deleteAll()
     }
+    func testMakeDictionary() {
```
Comment (cbaker6) [inline Tests/ParseSwiftTests/ParseFacebookCombineTests.swift:87]: Can you move this test to ParseFacebookCombineTests

**Review by cbaker6 — CHANGES_REQUESTED** [review]

File: Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift (hunk 1 above)
Comment (cbaker6) [reply Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift:47]: Can you also make the similar change to Twitter: https://github.com/parse-community/Parse-Swift/blob/252df53eee6e8f95178eb3f58ce7124ab5673b3a/Sources/ParseSwift/Authentication/3rd%20Party/ParseTwitter.swift#L27-L39

File: Tests/ParseSwiftTests/ParseFacebookCombineTests.swift (hunk 2 above)
Comment (dblythy) [reply Tests/ParseSwiftTests/ParseFacebookCombineTests.swift]: I ended up removing this because it's already covered by test (just the keys are wrong)

**Comment by cbaker6** [comment]
Can you add a change log entry?

**Review by cbaker6 — APPROVED** [review]

**Comment by cbaker6** [comment]
LGTM thanks for the fixes!

**Comment by dblythy** [comment]
Thank you @cbaker6!

**Pull request merged**